sys/shell/commands/sc_nimble_netif.c

```diff
@@ -30,17 +31,37 @@
 #include "net/bluetil/ad.h"
 #include "net/bluetil/addr.h"
 
-#if !IS_USED(MODULE_NIMBLE_AUTOCONN) && !IS_USED(MODULE_NIMBLE_STATCONN)
+#define FULL_CONTROL            !IS_USED(MODULE_NIMBLE_AUTOCONN) && \
+                                !IS_USED(MODULE_NIMBLE_STATCONN) && \
+                                !IS_USED(MODULE_NIMBLE_RPBLE)
```

Comment:
So how many different scenarios do we have here to test:

- autoconn
- statconn
- rple_ble
- non?

Reply:
- gnrc_networking (for regression bugs)
- tests/nimble_netif_ext
- tests/nimble_rpble_gnrc + rpble_ext
- tests/nimble_statconn_grnc + statconn_ext
- tests/nimble_autoconn_gnrc + autoconn_ext

Reply:
thanks I'll have to switch out for now, but I'll do some thorough testing tomorrow